<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugDialog</class>
 <widget class="QDialog" name="DebugDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>640</width>
    <height>480</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Debug Console</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/ChartGeany.qrc">
    <normaloff>:/png/images/icons/PNG/Bug.png</normaloff>:/png/images/icons/PNG/Bug.png</iconset>
  </property>
  <widget class="QPlainTextEdit" name="debugPlainTextEdit">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>2</y>
     <width>104</width>
     <height>70</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="undoRedoEnabled">
    <bool>false</bool>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="buttonFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>120</y>
     <width>320</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>320</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hideButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Hide</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../resources/ChartGeany.qrc"/>
 </resources>
 <connections/>
</ui>
